src/js/grapl-cdk/lib/engagement.ts

```diff
@@ -122,7 +126,7 @@ export class EngagementEdge extends cdk.NestedStack {
 
         this.event_handler = new lambda.Function(this, 'Handler', {
             runtime: lambda.Runtime.PYTHON_3_7,
-            handler: `engagement_edge.app`,
+            handler: `src.engagement_edge.app`,
```

Comment:
This is due to my making this Chalice app >1 file, and including the `__init__.py` in the `src` dir.